src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -69,14 +69,23 @@ const noNotebooksMessage = div({ style: { fontSize: 20 } }, [
   div([
     'To get started, click ', span({ style: { fontWeight: 600 } }, ['Create a New Notebook'])
   ]),
-  div({ style: { marginTop: '1rem', fontSize: 16 } }, [
+  div({ style: { margin: '1rem 0 1rem 0', fontSize: 16 } }, [
     h(Link, {
       ...Utils.newTabLinkProps,
       href: `https://support.terra.bio/hc/en-us/sections/360004143932`
     }, [`What's a notebook?`])
   ])
 ])
 
+const activeFileTransferMessage = div({
+  style: _.merge(
+    Style.elements.card.container,
+    { backgroundColor: colors.warning(0.15), flexDirection: 'none', justifyContent: 'start' })
+}, [
+  icon('warning-standard', { size: 19, style: { color: colors.warning(), flex: 'none', marginRight: '0.5rem', marginLeft: '-0.5rem' } }),
+  'Copying 1 or more notebooks from another workspace. ', span({ style: { fontWeight: 'bold' } }, ['This may take a few minutes.'])
```

Comment:
to reviewer: How do I prevent this whitespace after the first sentence from getting clobbered? I tried a bunch of things but clearly I did not try the right thing. Hopefully it's embarrassingly easy to fix

Reply:
Wrapping both the "Copying..." sentence and the bold span in another span works. I think the explanation for this is that wrapping them in a span puts both the text element and bold span in an inline formatting context, whereas currently they're in the outer `div`'s flex formatting context. Whitespace is handled differently depending on the formatting context.

https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace#html_largely_ignores_whitespace
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout/Intro_to_formatting_contexts

Alternatively, you could add a left margin on the bold span to create some space. `ch` units can be useful for cases like this.

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths

Reply:
Thank you!